integration_tests/features/support/steps.js

```diff
@@ -675,6 +675,32 @@ Then(/wallet (.*) detects all transactions as Mined_Confirmed/, {timeout: 1200*1
     }
 });
 
+Then(/while mining via (.*) all transactions in wallet (.*) are found to be Mined_Confirmed/, {timeout: 1200*1000}, async function (mmProxy, walletName) {
```

Comment:
```suggestion
Then(/while mining via (.*) all transactions in wallet (.*) are found to be Mined_Confirmed/, {timeout: 3800*1000}, async function (mmProxy, walletName) {
```

Reply:
My test timed out with the lesser value. This issue is more pronounced with more transactions. Upping the value as suggested worked fine for this 400 odd test.